pr-deploy/src/github.ts

```diff
@@ -21,11 +21,13 @@ type ChecksGetResponseData = Types['checks']['get']['response']['data'];
 type ChecksListForRefParams = Types['checks']['listForRef']['parameters'];
 type ChecksUpdateParams = Types['checks']['update']['parameters'];
 
-const ACTIONS: ChecksUpdateParams['actions'] = [{
-  label: 'Create a test site',
-  description: 'Serves the minified output of this PR.',
-  identifier: 'deploy-me-action',
-}];
+const ACTIONS: ChecksUpdateParams['actions'] = [
+  {
+    label: 'Create a test site',
+    description: 'Serves the minified output of this PR.',
```

Comment:
Since we now have multiple types of minified output (nomodule, module, experiment), maybe specify that this is nomodule?
```suggestion
    description: 'Serves the nomodule minified output of this PR.',
```